xrpl4j-core/src/test/java/org/xrpl/xrpl4j/model/transactions/CurrencyAmountTest.java

```diff
@@ -96,16 +110,23 @@ public void handleIssuance() {
 
     issuedCurrencyAmount.handle(
       ($) -> fail(),
-      ($) -> assertThat($.value()).isEqualTo("100")
+      ($) -> assertThat($.value()).isEqualTo("100"),
+      ($) -> fail()
     );
 
-    // null xrpCurrencyAmountHandler
-    assertThrows(NullPointerException.class, () ->
-      issuedCurrencyAmount.handle(null, ($) -> new Object())
-    );
-    // null issuedCurrencyAmountConsumer
-    assertThrows(NullPointerException.class, () ->
-      issuedCurrencyAmount.handle(($) -> new Object(), null)
+  }
+
+  @Test
+  public void handleMptAmount() {
```

Comment:
Do we need a dedicated test called `MptCurrencyAmountTest` (sort of like [XrpCurrencyAmountTest](https://github.com/XRPLF/xrpl4j/blob/main/xrpl4j-core/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java))?

Reply:
We have `MpTokenAmountTest`. Is that what you were looking for or are there other tests you think we should include?

Reply:
Yeah, that's what I was looking for -- the naming threw me off. See my other comment.